doc/Extensions/Applications.md

```diff
@@ -168,6 +168,24 @@ extend memcached /usr/local/bin/memcached
 4. Restart snmpd on your host
 5. On the device page in Librenms, edit your host and check `Memcached` under the Applications tab.
 
+### Unbound
+
+##### Agent
+[Install the agent](#agent-setup) on this device if it isn't already and copy the `unbound.sh` script to `/usr/lib/check_mk_agent/local/`
+
+Unbound configuration:
+
+```text
+# Enable extended statistics.
+server:
+        statistics-interval: 0
+        extended-statistics: yes
+        # set to yes if graphing tool needs it
```

Comment:
It's from the oficial guide [Unbound: How to Statistics ](https://www.unbound.net/documentation/howto_statistics.html), fixed on 8e7d40c